src/lib/spinel/multi_frame_buffer.hpp

```diff
@@ -451,6 +447,38 @@ template <uint16_t kSize> class MultiFrameBuffer : public FrameWritePointer
         kHeaderSize              = sizeof(uint16_t) + sizeof(uint16_t),
     };
 
+    template <typename Type, uint16_t kArrayLength> inline Type *GetArrayEnd(Type (&aArray)[kArrayLength])
```

Comment:
Can remove all `inline` here and below?
Any method implemented within a class declaration is implicitly considered as `inline`